### ISSUE TITLE
Updated Tutorial

### ISSUE THREAD

**Issue by ralpht** [lambdazen/pixy#3]
Is there an updated tutorial to work with gremlin console 3.3.3?

When I execute:
GremlinGroovyPipeline.class.metaClass.pixy { PixyTheory pt, String query, Object... args -> pt.makePipe(query, args).pixyStep(delegate) }

I get the following error:
No such property: GremlinGroovyPipeline for class: groovysh_evaluate

I am currently using:
bitsy version: 3.0.3-SNAPSHOT
pixy version:'3.0.0-SNAPSHOT

**Comment by lambdazen** [comment]
Yes, I dropped the ball on this. There are a bunch of name changes in TP3 that need to be documented, like GremlinPipeline was changed to GraphTraversal, and *Pipe was changed to *Step.

I will try to update the tutorial within the next week or two. 

Regards,

Sridhar.
